explainer.md

```diff
@@ -66,11 +62,11 @@ Following the code example above, if we have the following snippet of HTML:
 <p>Some |text|</p>
 ```
 
-where 'text' is covered by a Range (as denoted by the ```|``` characters) in the HighlightsMap under the 'example-highlight' identifier. In this case, during painting, the inline box containing ```Some text``` will detect that there is a Range that spans part of the box. Due to this, painting ```text``` will reference the Range and its association in the HighlightsMap in order to determine what styles to use. For this example the Range belongs to the 'example-highlight' identifier, which applies ```background-color:yellow``` and ```color:blue```, based on the map that was cascaded onto the ```<p>``` element. If the Range had an inline style of ```color:black```, this will be applied and overwrite the cascaded blue color for 'example-highlight'. The text 'Some ' will be painted as it normally would.
+where 'text' is covered by a Range (as denoted by the ```|``` characters) in the HighlightsMap under the 'example-highlight' identifier. In this case, during painting, the inline box containing ```Some text``` will detect that there is a Range that spans part of the box. Due to this, painting ```text``` will reference the Range and its container  HighlightsRangeGroup in the HighlightsMap in order to determine what styles to use. For this example the Range belongs to the 'example-highlight' identifier, which applies ```background-color:yellow``` and ```color:blue```, based on the map that was cascaded onto the ```<p>``` element. If the HighlightsRangeGroup had an inline style of ```color:black```, this will be applied and overwrite the cascaded blue color for 'example-highlight'. The text 'Some ' will be painted as it normally would.
 
-There can be multiple such ranges for a given inline box and Ranges added to the map can overlap &mdash; in these cases, the associated text will be split into a set of offsets, such that each member of the set has a unique collection of ranges covering it. The style properties are then computed for each member in the set by applying the styles of the applicable Ranges in ascending priority order (based on the ```priority``` property on Range), where the last write of a given property wins. In the event that Ranges overlap and have the same priority, the timestamp of when the Range was added to the map is used.
+There can be multiple such ranges for a given inline box and HighlightsRangeGroups added to the map can overlap &mdash; in these cases, the associated text will be partitioned into a set of intervals, such that each member of the set has a unique collection of HighlightsRangeGroups covering it. The style properties are then computed for each member in the set by applying the styles of the applicable HighlightsRangeGroups in ascending priority order (based on the ```priority``` property), where the last write of a given property wins. In the event that HighlightsRangeGroups overlap and have the same priority, the timestamp of when the HighlightsRangeGroup was added to the map is used.
```

Comment:
I think we agreed on the name HighlightRangeGroup (no s). Can you make sure it's consistent throughout the document?

Reply:
Oh, yes! My bad.